Makefile

```diff
@@ -30,3 +38,15 @@ generate_manifest:
 .PHONY: generate_schemas
 generate_schemas:
 	./generate_schemas.py definitions/Eiffel*Event/*.yml
+
+.PHONY: validate_examples
+validate_examples:
+	./examples/validate.py
+
+.PHONY: run_tests
+run_tests:
+	pytest --ignore=test_jsonformat.py
+
+.PHONY: json_format_test
+json_format_test:
+	pytest test_jsonformat.py
```

Comment:
Why duplicate what's already in tox.ini?

Reply:
Did find a good way to run only those commands but with some fresh eyes I have changed in in [b99e5e6](https://github.com/eiffel-community/eiffel/pull/366/commits/b99e5e6dda93ef189a4c5b4ee566cf9db5dcdb00)